BAZProjectC1/List/Model/Movie.swift

```diff
@@ -21,3 +21,31 @@ struct MenuRow {
     let detail: String
     let image: String
 }
+
+struct ResultParser : Decodable {
+    var page: Int?
+    var results: [MovieUpdate]?
+    var totalPages: Int?
+    var totalResults: Int?
+}
```

Comment:
ResultParser no es un nombre muy descriptivo, si bien es algo genérico por el momento está ligado a resultados de películas. Un mejor nombre sería `MoviesResultParser` por ejemplo.

Cuida los espacios:

```suggestion
struct MoviesResultParser: Decodable {
    var page: Int?
    var results: [MovieUpdate]?
    var totalPages: Int?
    var totalResults: Int?
}
```